internal/consumer/dlqMetadata.pb.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
how is this file generated ? Can you add a codgen target to the Makefile that can be used to update the generated file when the proto file changes ?

Reply:
It requires the person updating to install the protobuf compiler binary distributed by Google. I think we can leave it out for now.